src/test/java/AmazonTest.java

```diff
@@ -49,6 +50,30 @@ public void amazonTestOfPercentFunctionality() {
 
     }
 
+    @Test
+    public void checkPaginationAndHoveringOnBestSeller() {
+        CommonActions.createDriver(Config.PLATFORM_AND_BROWSER);
+        HomePage homePage = new HomePage(CommonActions.getDriver());
+        homePage.goTo();
+        homePage.waitForPageLoad();
+        homePage.chooseTopicButton();
+        homePage.waitForPreviousCategoryItemVisibility();
+        homePage.chooseCategory1("Smart Home");
+        homePage.waitForPreviousTopicItemVisibility();
+        homePage.chooseType("Smart Home Lighting");
+        SearchResultPage searchResultPage = new SearchResultPage(CommonActions.getDriver());
+        searchResultPage.waitForPageLoadVisibilityOfAllItem();
+        searchResultPage.seeAllResult();
+        AllSearchResultPage allSearchResultPage = new AllSearchResultPage(CommonActions.getDriver());
+        allSearchResultPage.waitForPageLoad();
+        SoftAssert softAssert = new SoftAssert();
+        softAssert.assertTrue(allSearchResultPage.textVisibilityWhenClickOnBestSellerButton());
+        softAssert.assertEquals(allSearchResultPage.numberOfItemsAcrossPages(), allSearchResultPage.itemsCountInPage());
+        softAssert.assertAll();
+
+
+    }
+
     @AfterMethod(alwaysRun = true)
     public void close() {
         if (Config.HOLD_BROWSER_OPEN) {
```

Comment:
So what is you written, is "If Hold browser open is true, then close the browser".
Does this sound correct to you? Fix the condition.